docs/proton-kafka.md

```diff
@@ -581,7 +589,10 @@ SETTINGS type='kafka',
          properties='message.max.bytes=1000000;message.timeout.ms=6000'
 ```
 
-Please note, not all properties in [librdkafka](https://github.com/confluentinc/librdkafka/blob/master/CONFIGURATION.md) are supported. The following ones are accepted in Proton today. Please check the configuration guide of [librdkafka](https://github.com/confluentinc/librdkafka/blob/master/CONFIGURATION.md) for details.
+This example sets the maximum Kafka message size to 1MB and the message timeout to 6 seconds.
+
+Please note while most configuration properties from `librdkafka` are supported, Timeplus may restrict or ignore certain settings. Here is the list of supported settings.
```

Comment:
the long table below now below to the Example section, since this is the only section under `## Properties for Kafka client {#advanced_settings}`. Better to add a new subsection